### PR TITLE
Update gradle versions.

### PR DESCRIPTION
**Pull request by IgorGanapolsky**

### CODE REVIEW

**Comment by ryanseys** [comment]
Ah this change from frame layout to merge was recent. Sorry I didn't notice that it caused a crash. I will take a look today into this PR in order to get it resolved. It shouldn't require an update to gradle though

File: build.gradle [hunk 1]
```diff
@@ -5,7 +5,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.3.0'
+        classpath 'com.android.tools.build:gradle:2.0.0-alpha1'
```
Comment (ryanseys) [inline build.gradle]: We don't want to depend on alpha versions in this sample

Comment (IgorGanapolsky) [inline build.gradle]: Why not use latest versions of Gradle?  Especially when Android Studio is
recommending it.

Comment (tmack8001) [inline build.gradle]: Your Android Studio is recommending it because you are on a pre-release version of Android Studio.

Comment (IgorGanapolsky) [inline build.gradle]: So is that a bad thing that shouldn't be trusted by Google?  I don't get it.

Comment (tmack8001) [inline build.gradle]: This is a beta/alpha release of software. Like all software that is in a pre-release state there could be bugs that haven't been addressed yet. See the [release announcement](http://android-developers.blogspot.com/2015/11/android-studio-20-preview.html) for more info.

The purpose of these sample repositories are to give developers example code, code style, and patterns to use in their production applications. It is VERY bad practice to put in pre-release dependencies into your production applications.

Comment (IgorGanapolsky) [inline build.gradle]: Umm... and using gradle-2.4 is good practice?

Comment (tmack8001) [inline build.gradle]: I'm just from the community... to get further clarification ask @ryanseys

Comment (ryanseys) [inline build.gradle]: Gradle 2.4 is fine and suits the needs of this project. We will update it when we feel it is necessary. Thanks for your contributions.

**Comment by ryanseys** [comment]
Fixed in 626f223d6d34abcf581bd7c09f225b9aba0ab3d1

**Pull request closed without merging**